Tests/test_SeqIO.py

```diff
@@ -488,6 +513,12 @@ def check_simple_write_read(records, indent=" "):
         # location
         warnings.simplefilter("ignore", BiopythonParserWarning)
 
+        # e.g. WARNING: Chain C is discontinuous at line 2645
+        warnings.simplefilter("ignore", PDBConstructionWarning)
+
+        # e.g. First line is not a 'HEADER'; can't determine PDB ID
+        warnings.simplefilter("ignore", BiopythonWarning)
```

Comment:
That looks like the warning ought to be ``BiopythonParserWarning`` instead? Another pre-existing minor issue.

Reply:
I agree -- I changed `Bio.SeqIO.PdbIO` to raise a `BiopythonParserWarning` instead in 41f310c.